python/basix/ufl_wrapper.py

```diff
@@ -1,48 +1,60 @@
 """Functions to directly wrap Basix elements in UFL."""
 
-from ufl.finiteelement.finiteelementbase import FiniteElementBase
-import hashlib
-import basix
+import ufl as _ufl
+from ufl.finiteelement.finiteelementbase import FiniteElementBase as _FiniteElementBase
+import hashlib as _hashlib
+import basix as _basix
+import typing as _typing
 
 
-class BasixElement(FiniteElementBase):
+class BasixElement(_FiniteElementBase):
     """A wrapper allowing Basix elements to be used with UFL."""
 
-    def __init__(self, element):
+    def __init__(self, element: _basix.finite_element.FiniteElement):
         super().__init__(
             element.family.name, element.cell_type.name, element.degree, None, tuple(element.value_shape),
             tuple(element.value_shape))
 
-        if element.family == basix.ElementFamily.custom:
-            self._repr = f"custom Basix element ({compute_signature(element)})"
+        if element.family == _basix.ElementFamily.custom:
+            self._repr = f"custom Basix element ({_compute_signature(element)})"
         else:
             self._repr = (f"Basix element ({element.family.name}, {element.cell_type.name}, {element.degree}, "
                           f"{element.lagrange_variant.name}, {element.dpc_variant.name}, {element.discontinuous})")
         self.basix_element = element
 
-    def mapping(self):
-        """Get the map type for this element."""
-        return map_type_to_string(self.basix_element.map_type)
+    def mapping(self) -> str:
```

Comment:
I think `map_type` would be better name, or does it need to match UFL?

Reply:
Needs to match ufl